lib/multi-acc-new-eks-mixed-observability-pattern/CreateIAMRoleNestedStack.ts

```diff
@@ -0,0 +1,48 @@
+import { NestedStack, NestedStackProps } from 'aws-cdk-lib';
```

Comment:
All our filenames are all sentence cases with dash and not mixed cases.

Reply:
done